.github/workflows/linux_build_matrix.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     with:
       jobName: "with LTO"
       compiler: gcc
-      cmakeflags: "-DLTO_PCSX2_CORE=ON"
+      cmakeflags: "" # "-DLTO_PCSX2_CORE=ON"
```

Comment:
Why are you disabling LTO?

Reply:
We were having some real bad linker issues with it earlier.

Reply:
It works fine for me (with clang at least).

Reply:
this is building with gcc, not clang. linker causing breakage with the qt translation.